### PR TITLE
Vertical Authorization Touch-ups

### LINKED ISSUE #266: Develop Testing for Vertical Bypassing Authorization Schema
**What would you like added?**

Further develop [Testing for Vertical Bypassing Authorization Schema](https://github.com/OWASP/wstg/blob/master/Testing_for_Vertical_Bypassing_Authorization_Schema_WSTG-AUTHZ-00X.md) and ensure it meets the [article standard](https://github.com/OWASP/wstg/blob/master/template/999.2_Template_Explanation_WSTG-FOO-002.md).

### PR DESCRIPTION
**Pull request by ThunderSon**
This PR closes #266 

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**
- Various touch-ups

I feel like changing the `Example 1` and `Example 2` to proper headings. What do you think? I don't enjoy this structure across the whole WSTG.

### CODE REVIEW

File: Testing_for_Vertical_Bypassing_Authorization_Schema_WSTG-AUTHZ-00X.md [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
+
+A Vertical authorization bypass is specific to the case that an attacker obtains a role higher than their own. This kind of test focuses on verifying how the Vertical authorization schema has been implemented for each role in getting access rights to data and resources of other users with different roles or privileges. Vertical authorization bypass may occur when an attacker gets access to more resources or data than they are allowed. For every function, page, specific role, and request that the application executes during the post-authentication phase, it is necessary to verify:
 
 - Is it possible to access resources that should be accessible only to higher role user?
 - Is it possible to operate functions on resources that should be operative only by a user that holds a higher or specific role identity?
 
 ## How to Test
 
-If you want to test manually, the process of testing for bypass authorization scheme could look like this:
+The process of testing for bypass authorization scheme follows the below:
```
Comment (kingthorin) [inline Testing_for_Vertical_Bypassing_Authorization_Schema_WSTG-AUTHZ-00X.md]:
```suggestion
The process of testing for bypass authorization scheme follows:
```

File: Testing_for_Vertical_Bypassing_Authorization_Schema_WSTG-AUTHZ-00X.md [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
+It is common in modern enterprises to define system roles to manage users and authorization to system resources. The purpose of the system role is to distinguish between user levels and to bind the assigned privileges to each role. An Authorization bypass enables an attacker to gain access to resources that are usually protected from an application or user.
+
+A Vertical authorization bypass is specific to the case that an attacker obtains a role higher than their own. This kind of test focuses on verifying how the Vertical authorization schema has been implemented for each role in getting access rights to data and resources of other users with different roles or privileges. Vertical authorization bypass may occur when an attacker gets access to more resources or data than they are allowed. For every function, page, specific role, and request that the application executes during the post-authentication phase, it is necessary to verify:
```
Comment (victoriadrake) [inline Testing_for_Vertical_Bypassing_Authorization_Schema_WSTG-AUTHZ-00X.md]:
```suggestion
Modern enterprises commonly define roles to manage users and authorization for system resources. Roles distinguish between user permission levels, each having certain privileges. An authorization bypass enables an attacker to gain access to permissions and resources that are assigned to other roles.

A vertical authorization bypass is specific to the case that an attacker obtains a role higher than their own. Testing for this bypass focuses on verifying how the vertical authorization schema has been implemented for each role. Vertical authorization bypass may occur when an attacker gets access to more resources or data than intended. For every function, page, specific role, and request that the application executes during the post-authentication phase, it is necessary to verify if it is possible to:
```

File: Testing_for_Vertical_Bypassing_Authorization_Schema_WSTG-AUTHZ-00X.md [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
+
+A Vertical authorization bypass is specific to the case that an attacker obtains a role higher than their own. This kind of test focuses on verifying how the Vertical authorization schema has been implemented for each role in getting access rights to data and resources of other users with different roles or privileges. Vertical authorization bypass may occur when an attacker gets access to more resources or data than they are allowed. For every function, page, specific role, and request that the application executes during the post-authentication phase, it is necessary to verify:
 
 - Is it possible to access resources that should be accessible only to higher role user?
 - Is it possible to operate functions on resources that should be operative only by a user that holds a higher or specific role identity?
 
 ## How to Test
 
-If you want to test manually, the process of testing for bypass authorization scheme could look like this:
+The process of testing for bypass authorization scheme follows:
```
Comment (victoriadrake) [inline Testing_for_Vertical_Bypassing_Authorization_Schema_WSTG-AUTHZ-00X.md]:
```suggestion
To test the possibility of vertically bypassing authorization schema:
```

File: Testing_for_Vertical_Bypassing_Authorization_Schema_WSTG-AUTHZ-00X.md [hunk 4]
```diff
@@ -1,21 +1,22 @@
...
+It is common in modern enterprises to define system roles to manage users and authorization to system resources. The purpose of the system role is to distinguish between user levels and to bind the assigned privileges to each role. An Authorization bypass enables an attacker to gain access to resources that are usually protected from an application or user.
+
+A Vertical authorization bypass is specific to the case that an attacker obtains a role higher than their own. This kind of test focuses on verifying how the Vertical authorization schema has been implemented for each role in getting access rights to data and resources of other users with different roles or privileges. Vertical authorization bypass may occur when an attacker gets access to more resources or data than they are allowed. For every function, page, specific role, and request that the application executes during the post-authentication phase, it is necessary to verify:
 
 - Is it possible to access resources that should be accessible only to higher role user?
 - Is it possible to operate functions on resources that should be operative only by a user that holds a higher or specific role identity?
```
Comment (victoriadrake) [inline Testing_for_Vertical_Bypassing_Authorization_Schema_WSTG-AUTHZ-00X.md]:
```suggestion
- Access resources that should be accessible only to a higher role user.
- Operate functions on resources that should be operative only by a user that holds a higher or specific role identity.
```

File: Testing_for_Vertical_Bypassing_Authorization_Schema_WSTG-AUTHZ-00X.md [hunk 5]
```diff
@@ -1,21 +1,22 @@
...
-If you want to test manually, the process of testing for bypass authorization scheme could look like this:
+The process of testing for bypass authorization scheme follows:
 
-1. For each role, register/generate a user.
-2. Generate and keep the session tokens by authenticating the application (one session token for each user).
+1. For each role, register a user.
+2. Generate and keep the session tokens by authenticating to the application (one session token for each role).
 3. For every request, change the session token from the original token to another role session token and diagnose the responses for each token.
-4. An application will be considered vulnerable if the responses are the same, contain same private data or indicate successful operation on other users resource or data.
+4. An application will be considered vulnerable if the responses are the same, contain same private data, or indicate successful operation on other users resource or data.
```
Comment (victoriadrake) [inline Testing_for_Vertical_Bypassing_Authorization_Schema_WSTG-AUTHZ-00X.md]:
```suggestion
4. An application will be considered vulnerable if the responses are the same, contain the same private data, or indicate successful operations on other user resources or data.
```

File: Testing_for_Vertical_Bypassing_Authorization_Schema_WSTG-AUTHZ-00X.md [hunk 6]
```diff
@@ -54,11 +55,15 @@ EventID=1000001
 Valid and legitimate response:
```
Comment (victoriadrake) [inline Testing_for_Vertical_Bypassing_Authorization_Schema_WSTG-AUTHZ-00X.md]:
```suggestion
The valid response:
```

**Comment by ThunderSon** [comment]
Just before this gets merged. Are we okay with `Example 1` and `Example 2`?

**Comment by kingthorin** [comment]
I'm okay with that.

**Pull request merged**